_scripts/memcached.js

```diff
@@ -0,0 +1,2 @@
+const spawn   = require('child_process').spawn;
+const memcachedProcess = spawn('memcached', ['-vv']);
```

Comment:
you need to pass an option here to inherit output from the spawn, `{ stdio: 'inherit' }` , see https://nodejs.org/api/child_process.html

Reply:
#### You goal here is to get the same output when running `./pm2 logs 16` and `memcached -vv` manually in the terminal:

![image](https://user-images.githubusercontent.com/128755/37314194-35c8caa8-2629-11e8-9926-e7e100715b68.png)